src/test/java/APICalls/PutCall.java

```diff
@@ -0,0 +1,28 @@
+package APICalls;
+
+import JacksonModels.Response;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+
+import java.io.IOException;
+
+public class PutCall extends BaseCall{
+
+    protected static String PUT_URL = "https://api.trello.com/1/boards/"+Id+"?key="+ apiKey+"&token="+ apiToken;
+    protected final CloseableHttpResponse httpResponse =  callPut();
+    protected final Response put = (Response) deserializeObject(httpResponse);
+
+    public PutCall() throws IOException {
+    }
+    protected static CloseableHttpResponse callPut() throws IOException {
+        try (CloseableHttpClient httpClient = HttpClients.createDefault();) {
+            HttpPut httpPut  = new HttpPut(PUT_URL);
```

Comment:
currently, HttpPut instance is created with just the URL
typically, PUT requests involve sending data, please, ensure the request is properly constructed with the required headers and body